backend/xivdpscalc/routing.py

```diff
@@ -52,23 +57,39 @@ def update_stats():
             pie = player_data['pie']
         )
     )
+
+    try:
+        rotation_data = data['rotation']
+        energy_drain_per_min = rotation_data.get('energyDrain', 4)
```

Comment:
I think it might make more sense to invert it and have this be non-ED per min, like lustrates per min or something.  Aetherflow stacks per min is non-uniform wrt time, and its impossible to burn 4 aetherflow stacks on minute 2 or smt.  This also brings it logically in line with adlo per min, conceptually as a "deviation from ideal" value.

Reply:
a problem i see with that is that non-ed per min is still non-uniform wrt time as you could in theory use dissipation and use all those stacks on healing too. 

i think that we should leave this for now since the sim that is currently used is still the naive 3 minute sim, but we should revisit the idea in your rewrite pr. this pr is meant to get us to feature parity with the google sheets so that we can release this for public beta.

Reply:
Ack. LGTM